### PR TITLE
Util: treeshaking tasks

### PR DESCRIPTION
**Pull request by ScottyPoi**
Util treeshaking task cleanup from #3560 

- `withdrawal.ts`
  - `Withdrawal` static constructors extracted to functions
- `requests.ts` renamed `request.ts`
  - static constructors extracted to functions
    - `WithdrawalRequest`
    - `ConsolidationRequest`
    - `DepositRequest`

### CODE REVIEW

**Comment by holgerd77** [comment]
In-between feedback: this looks good so far

**Comment by ScottyPoi** [comment]
> In-between feedback: this looks good so far

Ran out of time before the weekend.  Will finish this up today

File: packages/util/src/request.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
```
Comment (scorbajio) [inline packages/util/src/request.ts:1]: Could disabling no-use-before-define be avoided?

File: packages/util/src/withdrawal.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
```
Comment (scorbajio) [inline packages/util/src/withdrawal.ts:1]: Same thing here, could disabling no-use-before-define be avoided?

**Review by holgerd77 — DISMISSED** [review]
Some basic structural things to discuss respectively adjust.

**Comment by scorbajio** [comment]
Please update the example embeddings in the `util` docs and any other package docs that have modified examples from this PR. Also see [here](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3611#issuecomment-2310781004) for more details.

**Comment by holgerd77** [comment]
Can this be finalized so that we can merge?

File: packages/block/test/eip4895block.spec.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { RLP } from '@ethereumjs/rlp'
 import {
   Address,
   KECCAK256_RLP,
-  Withdrawal,
+  createWithdrawalFromValuesArray,
```
[the remark this answers is not in the document]
Comment (ScottyPoi) [reply packages/block/test/eip4895block.spec.ts]: :heavy_check_mark:  [2c368ea](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3589/commits/2c368ea66c67ce744b5d959d73ecc24c215c964f)

File: packages/util/src/request.ts [hunk 4]
```diff
@@ -249,32 +179,110 @@ export class ConsolidationRequest extends CLRequest<CLRequestType.Consolidation>
...
+    Uint8Array,
+  ]
+  return createDepositRequest({
+    pubkey,
+    withdrawalCredentials,
+    amount: bytesToBigInt(amount),
+    signature,
+    index: bytesToBigInt(index),
+  })
+}
```
[the remark this answers is not in the document]
Comment (ScottyPoi) [reply packages/util/src/request.ts:231]: addressed in [61354dc](https://github.com/ethereumjs/ethereumjs-monorepo/pull/3589/commits/61354dcf84cf7090a27b96e4e8f5573b145ddda3)

File: packages/util/src/withdrawal.ts (hunk 2 above)
Comment (ScottyPoi) [reply packages/util/src/withdrawal.ts]: :heavy_check_mark:

**Comment by ScottyPoi** [comment]
changes addressed.  ready for re-review

**Review by acolytec3 — APPROVED** [review]
LGTM

**Comment by holgerd77** [comment]
Nice, looks good! 🙂

**Pull request merged**